src/logsWebview.ts

```diff
@@ -106,7 +106,7 @@ export class LogsPanel extends WebPanel {
 						switch (message.command) {
 							case 'content':
 								const elt = document.getElementById('content');
-								const text = message.text.split('\\n');
+								const text = message.toString().split('\\n');
```

Comment:
what is the advantage of toString() versus getting the "text"? or is it that the retrieved message doesn't have the same structure when using kamel cli instead of kubectl?

Reply:
I was hitting a problem in the hosted javascript where it was throwing an error stating that split was not available on message.text. This was the workaround. I suspect what you are saying is true - that there's a difference in the structure of the message returned.